sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/Constants.java

```diff
@@ -11,14 +11,15 @@
 import java.util.List;
 
 public final class Constants {
-  public static String DEFAULT_SALESFORCE_API_VERSION = "53.0";
+  public static String DEFAULT_SALESFORCE_API_VERSION = "55.0";
```

Comment:
This needs to stay at `53.0`, as this is the default for users that didn't specify any version. This version will never change. Instead, we will eventually force all users to explicitly configure a default version.